src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/TransferClassificationActivityImpl.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassificationDao;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+@Lazy
+@Slf4j
+@Component
+public class TransferClassificationActivityImpl implements TransferClassificationActivity {
+    private final ClassificationDao classificationDao;
+
+    public TransferClassificationActivityImpl(ClassificationDao classificationDao) {
+        this.classificationDao = classificationDao;
+    }
+
+    @Override
+    public boolean classify(Long orgId, String iuv, String iur, int transferIndex) {
+        cleanUpCurrentProcessingRequests(orgId, iuv, iur, transferIndex);
+
+        return false;
+    }
+
+    private boolean cleanUpCurrentProcessingRequests(Long orgId, String iuv, String iur, int transferIndex) {
+        log.debug("Deleting classifications for organization id: {} and iuv: {}", orgId, iuv);
```

Comment:
The start of the process should be logged as INFO and moved on the classify method, this way this method will contain just 1 row, if we don't plan to add other lines, let's move in classify method